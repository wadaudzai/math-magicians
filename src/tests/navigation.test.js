import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/navigation';

describe('Navbar component testing using snapshots', () => {
  it('Test Navbar component renders', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Navbar />
        </BrowserRouter>,
      );
    expect(tree).toMatchSnapshot();
  });
});