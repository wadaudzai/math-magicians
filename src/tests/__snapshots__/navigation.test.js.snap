// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component testing using snapshots Test Navbar component renders 1`] = `
<nav
  className="navigation"
>
  <h1
    className="text-1"
  >
    Math Magicians
  </h1>
  <ul
    className="list"
  >
    <li>
      <a
        className="nav-link"
        href="/"
        onClick={[Function]}
      >
         Home 
      </a>
    </li>
    <li>
      <a
        className="nav-link"
        href="/Calculator"
        onClick={[Function]}
      >
         Calculator 
      </a>
    </li>
    <li>
      <a
        className="nav-link"
        href="/Quotes"
        onClick={[Function]}
      >
         Quotes 
      </a>
    </li>
  </ul>
</nav>
`;
